apps/web/playwright/booking-pages.e2e.ts

```diff
@@ -53,8 +52,8 @@ testBothBookers.describe("free user", (bookerVariant) => {
     // book same time spot again
     await bookTimeSlot(page);
 
-    // check for error message
-    await expect(page.locator("[data-testid=booking-fail]")).toBeVisible();
+    await page.locator("[data-testid=booking-fail]").waitFor();
```

Comment:
Was auto wait failing or why do we need to explicitly wait here?

Reply:
```suggestion
```